python/build.py

```diff
@@ -112,6 +117,10 @@ def get_python_executable():
 shutil.rmtree('taichi/tests')
 shutil.rmtree('taichi/examples')
 shutil.rmtree('taichi/assets')
+try:
+    os.remove('taichi/CHANGELOG.md')
+except:
```

Comment:
```suggestion
except FileNotFoundError:
```
mb, forgot to comment here earlier :p